### ISSUE TITLE
Add "highlights" section to profile page

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#1777]
Hi, this is a [first-timers-only issue](https://github.com/publiclab/plots2/projects/2). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### Problem

We're hoping to add a few more feature to the profiles page and also reorganize it for readability: https://publiclab.org/profile/warren

This is a multi-part issue, so if you can start with just creating the columns, we can help you through the next steps -- just open a pull request and we'll be there to help out!

### Solution

We need a 3-column space using Bootstrap columns just below the bio, to display some basic info about the user. See docs here: https://getbootstrap.com/docs/3.3/css/#grid-example-mixed, so for example:

```html
<div class="row">
  <div class="col-md-4">4 questions asked</div>
  <div class="col-md-4">2 answers posted</div>
  <div class="col-md-4">12 comments posted</div>
</div>
```

For example, columns could show:

* Number of questions posted (using `@profile_user.questions.count`, see [here](https://github.com/publiclab/plots2/blob/master/app/models/user.rb#L280))
* Number of answers posted (using probably `Answer.where(user_id: @profile_user.id).count`)
* Number of comments posted (using probably `Comment.where(uid: @profile_user.id).count`)

We may expand this out later, but this is a good starting point. The code could go onto the profile page template here:

https://github.com/publiclab/plots2/blob/master/app/views/users/profile.html.erb#L126

It could look basically like this:

![img_20171114_153905](https://user-images.githubusercontent.com/24359/32811222-6e051d96-c953-11e7-8807-cf3594c17a27.jpg)

## Steps to Fix

- [ ] claim this issue with a comment here, below, and ask any clarifying questions you need
- [ ] set up a repository locally following the README instructions, and make sure that all tests pass
- [ ] try to fix the issue following the steps above, but even before you're done, you can:
- [ ] commit your changes and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] alert someone via the developers list (see below) to have your pull request merged. This may take a few extra steps depending on when you do it, but we'll help you out!

Please email the developers list (see https://publiclab.org/wiki/developers) or go to [the chatroom](http://publiclab.org/chat) if you have questions, and take a look at [our first-timers landing page for more information](https://github.com/publiclab/plots2/projects/2)!

**Comment by Gauravano** [comment]
Hey @jywarren , I like to work on this issue

**Comment by jywarren** [comment]
Please do! Thank you!!

**Comment by jywarren** [comment]
This is complete! @Gauravano - when you open a PR, you can add the text `fixes #1234` and it'll auto-close the related issue. Great work on this, are you interested in another? Thanks!

**Comment by jywarren** [comment]
Awesome! And no need to call me "sir" please -- just Jeff will be fine.
Thank you!
